spec/predicates/deployment.md

```diff
@@ -0,0 +1,461 @@
+# Predicate type: Deployment
+
+Type URI: https://in-toto.io/attestation/deployment
+
+Version 1.0
+
+## Purpose
+
+To authoritatively express which environment an artifact is allowed to be deployed to.
```

Comment:
I think a bit more clarity would help here, because this in my reading is trying to make a _policy_ claim about where an artifact may be deployed to. To rephrase, I'm trying to understand here (and elsewhere in this document) why this doesn't end up being a de-facto policy. I know the lines can be a bit blurry with things like the VSA but I think it's currently clearer there.

Reply:
The first paragraph of the `Use cases` has `A deployment environment may be a cloud machine where we deploy containers, a developer workstation where we deploy packages (pip, npm, etc.) or even an Android device where we deploy applications.`

Maybe I should move this to the description? Wdut?

Reply:
How about a purpose like

"To authoritatively say that an artifact has been approved to be deployed in specific environments.  'Deployed in specific environments' may include a) running a workload on a Kubernetes cluster, b) installing a package on a workstation, c) installing an app on a smartphone.  This attestation expresses _both_ that the artifact has been 'approved' by the issuing party _as well as_ additional conditions (scopes) that the issuing party places on which environments the artifact may be deployed to.  It assumes that the issuing party and the parties managing the deployment environments are part of the same trust domain.  E.g. the issuing party assumes the environments will adhere to the conditions placed on deployment of the artifact.  The exact mechanism of how the environments will enforce these constraints is out-of-scope."

Separately (and optional!), would it help if, _for now_, this predicate type were limited to Cloud environments?  I think it's pretty clear how this can work in Cloud environments so if it met your current needs it might make it easier for everyone to understand if the purpose was less broad.  In the future the format could be amended to expand the types of environments it applies to.  WDYT?

Reply:
> "To authoritatively say that an artifact has been approved to be deployed in specific environments. 'Deployed in specific environments' may include a) running a workload on a Kubernetes cluster, b) installing a package on a workstation, c) installing an app on a smartphone. This attestation expresses both that the artifact has been 'approved' by the issuing party as well as additional conditions (scopes) that the issuing party places on which environments the artifact may be deployed to. It assumes that the issuing party and the parties managing the deployment environments are part of the same trust domain. E.g. the issuing party assumes the environments will adhere to the conditions placed on deployment of the artifact. The exact mechanism of how the environments will enforce these constraints is out-of-scope."

LGTM

Reply:
Changes made.